src/App.tsx

```diff
@@ -22,6 +22,7 @@ import {
 
 import Row from './components/Row';
 import Separator from './components/Separator';
+import Between from './components/Row Between';
```

Comment:
```suggestion
import RowBetween from './components/RowBetween';
```
We shouldn't have any component with space on the name

Reply:
the component name is `RowBetween`